ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -406,22 +395,15 @@ gst_tensor_src_tizensensor_class_init (GstTensorSrcTIZENSENSORClass * klass)
           DEFAULT_PROP_FREQ_N, DEFAULT_PROP_FREQ_D,
           G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
 
-#if (GST_VERSION_MAJOR == 1) && (GST_VERSION_MINOR >= 8) /* >= 1.8 */
-  gst_element_class_add_static_pad_template (gstelement_class, &src_factory);
-#elif (GST_VERSION_MAJOR == 1) /* 1.0 ~ 1.8 */
-  {
-    GstPadTemplate *pad_template;
-    GstCaps *pad_caps;
-    pad_caps = gst_caps_from_string (GST_TENSOR_CAP_DEFAULT
-        "; other/tensors, num_tensors = 1, framerate = " GST_TENSOR_RATE_RANGE);
-    pad_template = gst_pad_template_new ("src", GST_PAD_SRC, GST_PAD_ALWAYS,
-        pad_caps);
-    gst_element_class_add_pad_template (gstelement_class, pad_template);
-    gst_caps_unref (pad_caps);
-  }
-#else
-#error We support GStreamer 1.x only.
-#endif
+  /* pad template */
+  /** @todo Narrow down allowed tensors/tensor. */
+  pad_caps = gst_caps_from_string (GST_TENSOR_CAP_DEFAULT "; "
+      GST_TENSORS_CAP_WITH_NUM ("1"));
+  pad_template = gst_pad_template_new ("src", GST_PAD_SRC, GST_PAD_ALWAYS,
+      pad_caps);
+  gst_element_class_add_pad_template (gstelement_class, pad_template);
+  gst_caps_unref (pad_caps);
+
```

Comment:
Do you support GST 1.0~1.7 with this code as well? (Tizen 4.0 uses GST 1.6.1)

Reply:
just removed `#if (GST_VERSION_MAJOR == 1) && (GST_VERSION_MINOR >= 8)`, other code is same.

Reply:
Tested at https://build.tizen.org/package/show/devel:Tizen:6.0:AI:GHM_TZN4/nnstreamer